manifest/dft_unit_functionals.yaml

```diff
@@ -0,0 +1,168 @@
+# Local Density Approximation
+# TODO: add DOI
```

Comment:
This is work in progress, understandably, but it might be a good idea to add a quick explanation of the data contained in this file as a header.